client/packages/invoices/src/Prescriptions/api/hooks/useInsurance.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@openmsupply-client/common';
+import { usePrescriptionGraphQL } from '../usePrescriptionGraphQL';
+import { INSURANCE, LIST } from './keys';
+
+export const useInsurance = (
+  insuranceId: string | undefined | null,
+  storeId: string
```

Comment:
`apiModern` has a new hook called `usePatientGraphQl` that we can directly call in this file. Removing the need to pass it down as a prop

Reply:
Do you mean `usePrescriptionGraphQL` @GeronimoJohn ? I don't think I understand your comment...